go/gregor/storage/db.go

```diff
@@ -0,0 +1,54 @@
+package storage
+
+import (
+	"encoding/hex"
+
+	"github.com/keybase/client/go/gregor"
+	"github.com/keybase/client/go/libkb"
+)
+
+type localDismissalsRecord struct {
+	Dismissals [][]byte `json:"d"`
+}
+
+type LocalDb struct {
+	libkb.Contextified
+}
+
+func NewLocalDB(g *libkb.GlobalContext) *LocalDb {
+	return &LocalDb{
+		Contextified: libkb.NewContextified(g),
+	}
+}
+
+func dbKey(u gregor.UID) libkb.DbKey {
+	return libkb.DbKey{Typ: libkb.DBGregor, Key: hex.EncodeToString(u.Bytes())}
+}
+
+func dbKeyLocalDismiss(u gregor.UID) libkb.DbKey {
+	return libkb.DbKey{Typ: libkb.DBGregor, Key: "_ld" + hex.EncodeToString(u.Bytes())}
+}
+
+func (db *LocalDb) Store(u gregor.UID, state []byte, localDismissals [][]byte) error {
```

Comment:
Do you have any interest in versioning this?

Reply:
Not really, there is really no loss in having to blow it away.